packages/editor/src/components/editor-help/index.native.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import { View, Text } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { BottomSheet, PanelBody } from '@wordpress/components';
+import { useMemo } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { withPreferredColorScheme } from '@wordpress/compose';
+import {
+	helpFilled,
+	plusCircleFilled,
+	alignJustifyAlt,
+	trashFilled,
+	cogAlt,
+} from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import styles from './style.scss';
+import HelpDetailNavigationScreen from './help-detail-navigation-screen.js';
+import HelpTopicRow from './help-topic-row.js';
+
+const HELP_TOPICS = [
+	{ label: __( 'What is a block?' ), icon: helpFilled },
+	{ label: __( 'Add blocks' ), icon: plusCircleFilled },
+	{ label: __( 'Move blocks' ), icon: alignJustifyAlt },
+	{ label: __( 'Remove blocks' ), icon: trashFilled },
+	{ label: __( 'Customize blocks' ), icon: cogAlt },
+];
+
+function EditorHelpTopics( { isVisible, onClose, getStylesFromColorScheme } ) {
+	const bottomSheetHeaderTitleStyle = getStylesFromColorScheme(
+		styles.bottomSheetHeaderTitle,
+		styles.bottomSheetHeaderTitleDark
+	);
+
+	return useMemo( () => (
```

Comment:
@dcalhoun I noticed it's use in other areas of the app when dealing with bottom sheets. Do you think I should remove it?

Reply:
@AmandaRiu no, I have no concrete reason to avoid its usage. I asked due to my understanding is that it's generally best to avoid premature optimization. I was curious if there a specific performance issue we needed to address. 

If this pattern is present within other bottom sheets, there may have a very good reason. It could be worthwhile reviewing the history of those other bottom sheets at some point to understand why `useMemo` is present, but I am not requesting that you seek that out right now.

Reply:
I'll definitely research that more during this project and let you know what I find. For now, I'm leaving it as-is.